rabbitmq/meta/heka.yml

```diff
@@ -1,15 +1,53 @@
-{%- if pillar.rabbitmq.server is defined %}
-input:
-  logstreamer_rabbitmq:
-	type: "LogstreamerInput"
-	log_directory: "/var/log/rabbitmq"
-	file_match: 'rabbit@.+\.log\.?(?P<Index>\d+)?(.gz)?'
-	priority: ["^Index"]
-	decoder: "rabbitmq"
-	oldest_duration: "168h"
-decoder:
-  rabbitmq:
-    type: "SandboxDecoder"
-    filename: "lua_modules/decoders/rabbitmq.lua"
-    module_directory: "/usr/share/heka/lua_modules;/usr/share/heka/lua_modules/common"
-{% endif %}
+log_collector:
+  decoder:
+    rabbitmq_service:
+      engine: sandbox
+      module_file: /usr/share/lma_collector/decoders/rabbitmq.lua
+      module_dir: /usr/share/lma_collector_modules;/usr/share/heka/lua_modules
+      config:
+        include_full_notification: false
+  input:
+    rabbitmq_log_stream:
+      engine: logstreamer
+      log_directory: "/var/log/rabbitmq"
+      file_match: 'rabbit@(?P<Node>.+)\.log$'
+      differentiator: ["rabbitmq.", "Node"]
+      decoder: "rabbitmq_decoder"
+      splitter: "rabbitmq_splitter"
+  splitter:
+    rabbitmq_splitter:
+      engine: token
+      delimiter: "\n\n(=[^=]+====)"
+      delimiter_eol: false
+metric_collector:
+  filter:
```

Comment:
This filter and the one below will be generated from the alarms that will be defined in the user model. At least this is our current thinking. Obviously we need to flesh out the details and @elemoine is working on it.
So I would drop the metric_collector part for now and focus on the collectd plugins.